ui/src/notebooks/components/panel/NotebookPanel.tsx

```diff
@@ -54,6 +58,7 @@ const NotebookPanel: FC<Props> = ({index, children}) => {
           margin={ComponentSize.Small}
           justifyContent={JustifyContent.FlexEnd}
         >
+          {controls}
```

Comment:
does this conjure the feels you were looking for? it looks pretty lightweight

Reply:
yup yup